server/app/assets/javascripts/toggle.ts

```diff
@@ -1,4 +1,5 @@
 /** This class controls the toggling of toggle components. */
+import {addEventListenerToElements} from './util'
```

Comment:
nit: space makes things a little easier to read
```suggestion
import {addEventListenerToElements} from './util'

```